hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/SnapshotDeletingService.java

```diff
@@ -89,16 +86,17 @@ public class SnapshotDeletingService extends AbstractKeyDeletingService {
 
   private final OzoneManager ozoneManager;
   private final OmSnapshotManager omSnapshotManager;
-  private final SnapshotChainManager chainManager;
+  private final SnapshotChainManager snapshotChainManager;
```

Comment:
renaming variables for a patch is not a good idea. Increases Patch size footprint unnecessarily.

Reply:
done